src/routes/api/index.js

```diff
@@ -27,6 +27,10 @@ const listRoutesApi = [
     path: '/messages',
     route: require('./message.route'),
   },
+  {
+    path: '/cartDetails',
+    route: require('./CartDetail.route'),
+  },
```

Comment:
```suggestion
  {
    path: '/cart-details',
    route: require('./CartDetail.route'),
  },
```
- route ít khi người ta viết hoa. trên 2 từ viết theo kiểu tôi gợi ý